docs/generated/packages/js/generators/library.json

```diff
@@ -113,7 +113,7 @@
       "bundler": {
         "description": "The bundler to use.",
         "type": "string",
-        "enum": ["none", "esbuild", "rollup", "vite", "webpack"],
+        "enum": ["none", "esbuild", "rollup", "vite"],
```

Comment:
question: why are we removing webpack?

Reply:
I think we want to use `webpack` to build apps and use the alternatives (`esbuild` `rollup` & `vite`) for libraries.

Reply:
I don't understand why we cannot use `webpack` to build libs but I don't have any current libs built with `webpack` to back it up. We can add it back later but would we need a migration then?

Reply:
I am not sure if many subscribed to using `webpack` for libs. 
We could add a migration if necessary but I think the majority of the community used `webpack` for apps.
cc: @jaysoo